rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -111,7 +170,7 @@ def put(self, deployment_id, **kwargs):
             skip_plugins_validation=skip_plugins_validation,
             site=site,
             runtime_only_evaluation=request_dict.get(
-                'runtime_only_evaluation', False),
```

Comment:
Why remove the trailing comma here?

Reply:
Bu mistake I think while was doing rebase and merge conflict but this should be fixed